packages/scheduled-tasks/src/lib/strategies/ScheduledTaskRedisStrategy.ts

```diff
@@ -0,0 +1,71 @@
+import { container } from '@sapphire/framework';
+import type { Job, JobOptions, Queue, QueueOptions } from 'bull';
+import Bull from 'bull';
+import type { ScheduledTasks } from '../structures/ScheduledTask';
+import type { ScheduledTaskCreateRepeatedTask, ScheduledTasksTaskOptions } from '../types';
+import type { ScheduledTaskBaseStrategy } from '../types/ScheduledTaskBaseStrategy';
+
+export interface ScheduledTaskRedisStrategyOptions {
+	queue?: string;
+	bull?: QueueOptions;
+}
+
+export interface ScheduledTaskRedisStrategyJob {
+	task: string;
+	payload?: any;
+}
+
+export class ScheduledTaskRedisStrategy implements ScheduledTaskBaseStrategy {
+	public readonly options: QueueOptions;
+	public readonly queue: string;
+
+	private _client!: Queue;
```

Comment:
I don't really like using underscore prefixes for private fields. Please remove.